README.md

```diff
@@ -23,20 +25,35 @@ GordianCosigner-Catalyst is currently a late alpha. It should not be used for pr
 
 At current, this Catalyst repo is only used for our iOS release, but we hope to eventually use it for the MacOS release as well. (A security lock on the MacOS camera currently prevents us from doing so.)
 
-## Installation
+## Installation Instructions
 
 **Gordian Cosigner** is available for testing from Testflight [here](https://testflight.apple.com/join/sJTaoUsM).
 
-## Usage
+## Usage Instructions
+
+**Gordian Cosigner** supports the middle step of multisig signing, after a multisig has been Initiated, while it is being Cosigned, and before it is Finalized. Usually, it will be used by a cosigner holding one of the keys for a multisignature before handing it back to another user for finalization.
```

Comment:
"one of the keys for a multisignature before" — how about "one of the keys used in a multisig account transaction, before".

Reply:
Done.